### PR TITLE
License update for c formulae

### PR DESCRIPTION
**Pull request by chenrui333**
license updates for `c*` formulae

### CODE REVIEW

File: Formula/chibi-scheme.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class ChibiScheme < Formula
   url "https://github.com/ashinn/chibi-scheme/archive/0.8.tar.gz"
   sha256 "8a077859b123216c123c243db391b0fe4c0cf73978c7cdd7b8ea853a48192756"
   revision 1
+  license "BSD-3-Clause"
```
Comment (Rylan12) [inline Formula/chibi-scheme.rb:7]:
```suggestion
  license "BSD-3-Clause"
  revision 1
```
```
❯ brew style --display-cop-names homebrew/core
== chibi-scheme.rb ==
C:  7:  3: FormulaAudit/ComponentsOrder: license (line 7) should be put before revision (line 6)
```

File: Formula/cmake.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ class Cmake < Formula
   desc "Cross-platform make"
   homepage "https://www.cmake.org/"
   head "https://gitlab.kitware.com/cmake/cmake.git"
+  license "BSD-3-Clause"
```
Comment (Rylan12) [inline Formula/cmake.rb:5]:
```suggestion
  license "BSD-3-Clause"
  head "https://gitlab.kitware.com/cmake/cmake.git"
```
```
❯ brew style --display-cop-names homebrew/core
== cmake.rb ==
C:  5:  3: FormulaAudit/ComponentsOrder: license (line 5) should be put before head (line 4)
```

File: Formula/cmigemo.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@ class Cmigemo < Formula
   desc "Migemo is a tool that supports Japanese incremental search with Romaji"
   homepage "https://www.kaoriya.net/software/cmigemo"
   head "https://github.com/koron/cmigemo.git"
+  license "MIT"
```
Comment (Rylan12) [inline Formula/cmigemo.rb:5]:
```suggestion
  license "MIT"
  head "https://github.com/koron/cmigemo.git"
```
```
❯ brew style --display-cop-names homebrew/core
== cmigemo.rb ==
C:  5:  3: FormulaAudit/ComponentsOrder: license (line 5) should be put before head (line 4)
```

File: Formula/corsixth.rb [hunk 4]
```diff
@@ -4,6 +4,7 @@ class Corsixth < Formula
   url "https://github.com/CorsixTH/CorsixTH/archive/v0.64.tar.gz"
   sha256 "12389a95de0031baec1a3fc77208d44228177f49564f1c79ae763ab4aeeafa98"
   head "https://github.com/CorsixTH/CorsixTH.git"
+  license "MIT"
```
Comment (Rylan12) [inline Formula/corsixth.rb:7]:
```suggestion
  license "MIT"
  head "https://github.com/CorsixTH/CorsixTH.git"
```
```
❯ brew style --display-cop-names homebrew/core
== corsixth.rb ==
C:  7:  3: FormulaAudit/ComponentsOrder: license (line 7) should be put before head (line 6)
```

**Review by Rylan12 — CHANGES_REQUESTED** [review]

**Comment by chenrui333** [comment]
Nice catch, I should batch run the `brew style *` :)

**Comment by chenrui333** [comment]
Should be good now with `brew style`

**Review by Rylan12 — APPROVED** [review]

**Comment by SeekingMeaning** [comment]
Wait `curl` should be under the `curl` license: https://spdx.org/licenses/curl.html

**Comment by Rylan12** [comment]
@SeekingMeaning can you open a PR for this? As long as `brew style` passes it can be merged without CI runs (like this one)

**Pull request merged**